.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,39 @@
+name: Nightly
+on:
+  schedule:
+    # Times in UTC (PST+8), this translates to running Mon-Fri at midnight/1am (depending on DST)
+    - cron: 0 8 * * 1-5
+
+jobs:
+  plugins_test:
+    name: Nightly Plugin Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        linter-version: [Snapshots, Latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+        # TODO(Tyler): Determine if we want to wipe the download cache and start fresh each night.
```

Comment:
Do we want to wipe out the contents of the cache? Is the easiest way to do that just rm?

Reply:
Pretty much, yeah. Look at .github/actions/linux_build_and_test/action.yaml in the monorepo for an exaple.